docs/api/tray.md

```diff
@@ -217,9 +217,10 @@ Sets the `image` associated with this tray icon when pressed on macOS.
 
 Sets the hover text for this tray icon.
 
-#### `tray.setTitle(title)` _macOS_
+#### `tray.setTitle(title, [fontType])` _macOS_
 
 * `title` String
+* `fontType` String (optional) - The font family variant to display, one of `monospaced` (available in macOS 10.15+) or `monospacedDigit` (available in macOS 10.11+). When left blank, the title uses the default system font.
```

Comment:
Given there are a few things that could be customized here (font size comes to mind) to allow for easier API expansion maybe this should instead be a `fontOptions` object that has a single property `type`.  This would allow us to more easily add `size` or other attributes into this API without causing breaking changes or creating a list-of-parameters 😄